rules-reviewed/technology-usage/non-xml-rules-technology-usage.rhamt.xml

```diff
@@ -0,0 +1,353 @@
+<?xml version="1.0"?>
+<ruleset id="non-xml-technology-usage"
+         xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides statistical summaries of embedded 3rd party frameworks.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+        </dependencies>
+        <phase>PostMigrationRulesPhase</phase>
+    </metadata>
+    <rules>
+        <rule id="non-xml-technology-usage-01000">
+            <when>
+                <graph-query discriminator="TechnologyTagModel">
+                    <property name="name">EJB</property>
+                </graph-query>
+            </when>
+            <perform>
+                <technology-identified name="EJB">
+                    <tag name="Connect"/>
+                    <tag name="Bean"/>
+                    <tag name="Java EE"/>
+                </technology-identified>
+            </perform>
+        </rule>
+        <rule id="non-xml-technology-usage-02000">
+            <when>
+                <graph-query discriminator="TechnologyTagModel">
+                    <property name="name">EJB XML</property>
+                </graph-query>
+            </when>
+            <perform>
+                <technology-identified name="EJB XML">
+                    <tag name="Connect"/>
+                    <tag name="Bean"/>
+                    <tag name="Java EE"/>
+                </technology-identified>
+            </perform>
+        </rule>
+        <!--rule id="non-xml-technology-usage-03000">
```

Comment:
Why is this commented ? is not being used ? I see Enterprise Web Services in the spreadsheet

Reply:
I can't find anywhere where this tech tag is produced so although it was placed in the spreadsheet I've left the rule and test commented out. The other commented rules/tests are the same. We need to decide to remove them, first I think I need to discover why they are in the spreadsheet to ensure nothing has been missed.